style.css

```diff
@@ -0,0 +1,47 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+
+@media (max_width:460px)
+{
+    body {
+background-color: gray;
+color: white;
+    }
+}
+
+#nav-bar {
+    position: sticky;
+    top: 0%;
+    left: 0%;
+    z-index: 1;
+    padding:  1.5rem 3.5rem;
+    background-color: aqua;
+    box-shadow: 0 3px 5px rgba(red, green, blue, alpha);
+}
+ul {
+    list-style-type: none;
+    color: white;
+}
+a :hover {
+
+    text-decoration: none;
+    color: pink;
+}
+a{
```

Comment:
Please put a one line space

Reply:
- [ ] In this comment too please mention the reason why you ask for change.